src/WidenImage.js

```diff
@@ -0,0 +1,86 @@
+import './WidenImage.css'
+import utilities from './Utilities'
+
+// If enabled, widened images will have thin red dashed border
+const enableDebugBorders = false
```

Comment:
I'd only ever changed this manually, but honestly I think we should delete it. I had used it to make it _very_ apparent when an image had been widened by having it use loud colored borders, but I can just manually set such borders in `.wideImageOverride` when needed.